housekeeper/store/api/find.py

```diff
@@ -86,57 +93,63 @@ def tags(self) -> List:
 
     def file_(self, file_id: int):
         """Get a file by record id."""
-        return self.File.get(file_id)
+        return apply_file_filters(
+            files=self._get_file_query(),
+            filter_functions=[FileFilters.FILTER_BY_ID],
+            file_id=file_id,
+        ).first()
 
     def files(
-        self, *, bundle: str = None, tags: List[str] = None, version: int = None, path: str = None
+        self, bundle: str = None, tags: List[str] = None, version: int = None
     ) -> Iterable[File]:
         """Fetch files from the store."""
-        query = self.File.query
+        query = self._get_file_query()
         if bundle:
```

Comment:
The code below each if could be a function returning a query, you can also put the if into the function where it then starts with an if not bundle return simialrly to above

Reply:
I don't think it is good to wrap entire function bodies in if statements. But I'll extract the code below the if statements to separate functions.